### ISSUE TITLE
\Skautis\Exception\AuthenticationException

### ISSUE THREAD

**Issue by literat** [skaut/ws.skautis.cz#4]
Ahoj, brouzdám kódem a sem tam něco použiju. A narazil jsem na problém s \Skautis\Exception\AuthenticationException v AuthPresenter. Danou Exception totiž nemůžu najít. Nevím, jestli se nevztahuje ke Skautis knihovně jako takové, protože v jejím obsahu jsem nic takového nenalezl. A pokud použiji kód jako celek a vyvolám danou výjimku, tak jde aplikace na držku. Takže nevím, jestli není chyba tady anebo u mě.

**Comment by JindrichPilar** [comment]
Je mozne ze ws.skautis.cz byla napsana oproti starsi verzi skaut/Skautis, kde exceptions byly vsechny v jednom namespace. 

Pravdepodobne se jedna o exception co je nove https://github.com/skaut/Skautis/blob/2.x/src/Wsdl/AuthenticationException.php

**Comment by literat** [comment]
Jo, myslel jsem si, že to tak bude. Právě tuhle výjimku jsem našel. Takže tím pádem bych si dovolil tvrdit, že  je zde prostě jenom starý namespace \Skautis\Excepútion\AuthenticationException -> \Skautis\Wsdl\AuthenticationException.

**Comment by sinacek** [comment]
Jo to máš asi pravdu.
